src/main/java/com/github/fishio/settings/YamlSettingLoader.java

```diff
@@ -64,18 +64,19 @@ public YamlSettingLoader() {
 	 * 		the file to create.
 	 */
 	private void createSettingsFile(File file) {
```

Comment:
Can you explain the warning message on line 66 in the YamlSettingLoader.java

<review